examples/HexagonalTopology.ipynb

```diff
@@ -1137,10 +494,157 @@
     "ax.legend(handles=legend_elements, bbox_to_anchor=(0.1, 1.08), loc='upper left', \n",
     "          borderaxespad=0., ncol=3, fontsize=14)\n",
     "\n",
-    "plt.savefig('resulting_images/som_seed_hex.png')\n",
+    "plt.savefig('resulting_images/som_seed.png')\n",
     "plt.show()"
    ]
   },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Plotting: bokeh\n",
+    "Below, are plotting using bokeh to create an interactive hexagonal topology.\n",
+    "\n",
+    "> Note: Compared to matplotlib plot, this is rotated 90 degrees."
+   ]
+  },
+  {
+   "cell_type": "code",
```

Comment:
The output of this cell is not shown in the github preview. Can you add a link to the html page that renders the chart?

Reply:
Possibly using a service that renders the page without having to download it like this one: https://htmlpreview.github.io/

Reply:
That's a pretty cool service, thanks for the suggestion! I've tried to do it here. https://github.com/JustGlowing/minisom/pull/88/commits/750812ca6b4c30d53489b3f3381c63a10771475f

You won't be able to preview it immediately because the link is to the HTML file that will be on the `JustGlowing:master` branch when this PR gets merged. Currently, it exists on the `avisionh:example-hexagonaltopology-bokeh` branch.

Reply:
That said, I'm not sure if the HTML web renderer is working. 🙈 I've tested it by merging this branch into my forked version of this repo's `master` branch and tried to generate the plot. This is what I got:

https://htmlpreview.github.io/?https://github.com/avisionh/minisom/blob/master/examples/resulting_images/som_seed.html